src/plugins/deps/geomap-lib.js

```diff
@@ -3546,7 +3546,12 @@ MapLegend.prototype.symbolize = function( mapLayer ) {
 	}
 
 	// append the list to the legend
-	$( "#sb_" + layerName ).html( "<ul class='list-unstyled'>" + symbolList + "</ul>" );
+	if ( $( "#sb_" + layerName ).closest( ".wb-geomap" ).hasClass( "legend-label-only" ) && symbolList.match( /<li>/g ) && symbolList.match( /<li>/g ).length === 1 ) {
+		$( "#sb_" + layerName ).html( "<ul class='list-unstyled' role='presentation'>" + symbolList + "</ul>" );
```

Comment:
I suggest to add a comment explaining why we are setting the role presentation on the `<ul>` and `<li>` element